src/components/dashboard/Claim.js

```diff
@@ -348,13 +276,7 @@ const Claim = props => {
     }
   }
 
-  const handleNonCitizen = () => {
-    if (Config.claimQueue) {
-      handleClaimQueue()
-      return
-    }
-    handleFaceVerification()
-  }
+  const handleNonCitizen = () => handleFaceVerification()
```

Comment:
why not just `handleNonCitizen={handleFaceVerification}` in `<ButtonBlock/>`'s props ?